pkg/node/Dockerfile

```diff
@@ -6,4 +6,10 @@ ADD     node  node
 
 EXPOSE  9999
 
+RUN sed -i \
+        -e 's/udev_sync = 1/udev_sync = 0/' \
+        -e 's/udev_rules = 1/udev_rules = 0/' \
+        -e 's/obtain_device_list_from_udev = 1/obtain_device_list_from_udev = 0/' \
+        /etc/lvm/lvm.conf
```

Comment:
Could you please add some comments? I'm not really understand, what happens here :)

Reply:
Added the comments